lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -646,6 +646,17 @@ protected function prepare_links( $comment ) {
 			);
 		}
 
+		// Only grab one comment to verify the comment has children.
+		$comment_children = $comment->get_children( array( 'number' => 1, 'count' => true ) );
+		if ( ! empty( $comment_children ) ) {
+			$args = array( 'parent' => $comment->comment_ID );
+			$rest_url = add_query_arg( $args, rest_url( $this->namespace . '/' . $this->rest_base ) );
+
+			$links['children'] = array(
+				'href'       => $rest_url,
```

Comment:
@BE-Webdesign indentation is off after https://github.com/WP-API/WP-API/pull/2662/commits/b10fb736dff1ecb7addad4c91f24fccb6986c8a6